eQuiz/database/eQuiz.Database/create.sql

```diff
@@ -84,7 +84,8 @@ CREATE TABLE [dbo].[tblQuiz]
 	[InternetAccess] [BIT] NOT NULL,
 	[GroupId] [INT] NOT NULL,
 	CONSTRAINT [PK_tblQuiz_Id] PRIMARY KEY ([Id]),
-	CONSTRAINT [UK_tblQuiz_Name] UNIQUE ([Name])
+	CONSTRAINT [UK_tblQuiz_Name] UNIQUE ([Name]),
+	CONSTRAINT [CK_tblQuiz_StartDate_EndDate] CHECK ([StartDate] < [EndDate])
```

Comment:
In my opinion we should use the condition [StartDate] <= [EndDate].
What do you think?